src/modules/Lan.rb

```diff
@@ -703,11 +683,9 @@ def Import(settings)
       else
         NetworkService.use_wicked
       end
-      if Builtins.haskey(settings, "ipv6")
-        @ipv6 = Ops.get_boolean(settings, "ipv6", true)
-      end
 
-      LanItems.modified = true
+      @ipv6 = settings["ipv6"] || true
```

Comment:
well, it is. And not only in one file. But I'm not able to save them all now, so I'm only touching those directly affected by a PR.